doc/classes/Light.xml

```diff
@@ -51,6 +51,7 @@
 		</member>
 		<member name="light_size" type="float" setter="set_param" getter="get_param" default="0.0">
 			The size of the light in Godot units. Only considered in baked lightmaps and only if [member light_bake_mode] is set to [constant BAKE_ALL]. Increasing this value will make the shadows appear blurrier. This can be used to simulate area lights to an extent.
+			[b]Note:[/b] [member light_size] is not affected by [member Spatial.scale] (on the light itself or its parents).
```

Comment:
```suggestion
			[b]Note:[/b] [member light_size] is not affected by [member Spatial.scale] (The light's scale or its parent's scale).
```
I had to read that 4 times to figure out what it was saying, I think this should be more clear.